packages/codemirror-graphql/src/variables/__tests__/hint-test.js

```diff
@@ -32,10 +29,13 @@ function getHintSuggestions(query, variables, cursor) {
   const editor = createEditorWithHint(query);
   return new Promise(resolve => {
     const graphqlVariablesHint = CodeMirror.hint['graphql-variables'];
+
     CodeMirror.hint['graphql-variables'] = (cm, options) => {
+      console.log('\n\n\n\n\n\nHINT HIT\n\n\n\n\n');
```

Comment:
?

Reply:
sorry yeah, this PR is still very much a WIP if you see in the title